instance/schemas/theming.py

```diff
@@ -167,6 +166,27 @@ def ref(definition_name):
     "oneOf": [nullable_schema, theme_schema_v0, theme_schema_v1]
 }
 
+DEFAULT_THEME = {
+    "version": 1,
+    "main-color": "#126F9A",
+    "link-color": "#126F9A",
+    "header-bg": "#FFFFFF",
+    "footer-bg": "#FFFFFF",
```

Comment:
IMHO it's clearer to have "color" in the end of the name of all attributes which are colors (eg. header-bg-color, footer-bg-color).

Reply:
I agree with you, but there's not much we can do in the scope of this epic unless we want to do some variable name remapping (I'd avoid that).
The main reason for the variables names being like this is that they need to match the ones described on this repo: https://github.com/open-craft/edx-simple-theme/tree/guruprasad/wip-high-level-theme (we're using this simple theme).

Reply:
@alanoe, in the simple-theme that Giovanni has linked to, I have used variable names like `footer-bg`, `footer-color` etc. where the `bg` (the short form for "background") suffix refers to the background color and the `-color` suffix refers to the foreground/text color. This was done to create a distinction between the two and at the same time avoid the variable names from becoming too long.

Reply:
@lgp171188 @giovannicimolin Thanks for explaining the origin of those attributes names. IMHO bg-color and fg-color are more explicit and clearer than bg and color. Anyway, I guess we won't make such renamings improvements like this at this point, so never mind.